autobahn/asyncio/wamp.py

```diff
@@ -172,3 +172,106 @@ def create():
             loop.run_until_complete(protocol._session.leave())
 
         loop.close()
+
+
+# TODO - unify with previous class
+class ApplicationRunnerRawSocket(object):
+    """
```

Comment:
On the one hand, ``ApplicationRunner`` is a convenience API and needs improvement for more use-cases, but there should be just *one* way to run any WAMP connection; e.g. see https://github.com/crossbario/autobahn-python/issues/674 (and "LTS-API" tagged bugs).

On the other hand of course, there's not currently a way to ask ApplicationRunner to instantiate anything except WebSocket :/ 

The "real" solution is to make sure ``Component`` understands raw vs web socket. However, there is only a ``Component`` in the ``twisted/`` packages right now as that API isn't finalized (but, it currently does properly distinguish ws/tcp transports). I don't think this PR should be stalled/blocked on that though ... maybe we should just not change ApplicationRunner etc at all in this PR and do it separately?

For testing purposes if you have crossbar start your session, it can be configured with a raw-socket transport (with ``type: "rawsocket"`` vs ``type: "websocket"``).